docs/quick-ingestion-guides/snowflake/setup.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Setup
+---
+# Snowflake Ingestion Guide: Setup & Prerequisites
+
+In order to configure ingestion from Snowflake, you'll first have to ensure you have a Snowflake user with the `ACCOUNTADMIN` role or `MANAGE GRANTS` privilege and you know where to [execute queries](https://docs.snowflake.com/en/user-guide/ui-worksheet.html#) in Snowflake.
```

Comment:
This part is unclear to me:

"you know where to executed queries in Snowflake". I think this is unnecessary.